keyboards/alpha/keymaps/default/keymap.c

```diff
@@ -0,0 +1,82 @@
+#include "kb.h"
```

Comment:
I missed this. This should be replaced with `#include QMK_KEYBOARD_H` (or at least `#include "alpha.h"`).